public/humans.txt

```diff
@@ -1,3 +1,16 @@
 GOV.UK is built by a team at the Government Digital Service (with thanks to Martha!) in London,
-Bristol and Manchester.  If you'd like to join us, see https://jobs.jobvite.com/gds.
+Bristol and Manchester.  If you'd like to join us, see https://jobs.jobvite.com/gds
```

Comment:
Minor nitpick - end the sentence with a full stop?

Reply:
This was Helen Nickols' suggestion - I didn't actually ask why, but I guess it's to reduce noise when a paragraph ends with a URL.

Reply:
👍 cool